Add info about using env vars in Agent reference (#2016)
sensu/sensu-docs: content/sensu-go/5.16/reference/agent.md

{{< /language-toggle >}}

## Use environment variables with the Sensu agent

> After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to commands executed by Sensu resources, including handlers and checks.

Reviewer: ```suggestion
After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to check and hook commands executed by the Sensu agent.
```

I just remembered agents don't run handlers, so they arent a good example here.

{{< /language-toggle >}}

## Use environment variables with the Sensu agent

> After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to your Sensu resources.

Reviewer: ```suggestion
After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to commands executed by Sensu resources, including handlers and checks.
```

This includes your checks and plugins.

For example, if you configure a `SENSU_TEST_VAR` variable in your sensu-agent file, it will be available to use in your check configurations as `$SENSU_TEST_VAR`.

> The environment variables you provide in your sensu-agent file are also available for plugin processes.

Reviewer: ```suggestion
```

> The environment variables you provide in your sensu-agent file are also available for plugin processes.

Reviewer 2: This line seems redundant as on line 1312 it is mentioned that this works for plugins as well. Line 1314 is needed as it shows a more concrete example how to use them.

Reviewer 2: Resolving as it has been removed.
